src/ontology/imports/merged_import.owl

```diff
@@ -99565,11 +99574,11 @@ AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.ob
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0000734> "Disinhibition")
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0000734> <http://purl.obolibrary.org/obo/HP_5200241>)
 
-# Class: <http://purl.obolibrary.org/obo/HP_0000735> (Impaired social interactions)
+# Class: <http://purl.obolibrary.org/obo/HP_0000735> (obsolete Impaired social interactions)
```

Comment:
@matentzn are all the changes below because HPO had a release Mar 6?

Reply:
Yes - the import workflow will always update all imports to esnure that we have the latest versions of everything for mutual compatibility.